.packit.yaml

```diff
@@ -29,3 +29,11 @@ jobs:
     - fedora-all
     - epel-8
     - epel-7
+
+- job: tests
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-all
+    - epel-8
+    - epel-7
```

Comment:
Note that you don't need to specify both `tests` and `copr_build` jobs for pull requests unless you want to see separate test results for them. If only `tests` job is provided, copr builds are triggered automatically.

Reply:
If I'm not mistaking, recent updates require both test and copr_build to be defined